core/src/main/java/hudson/Functions.java

```diff
@@ -755,7 +755,7 @@ public static String nbspIndent(String size) {
         int i = size.indexOf('x');
         i = Integer.parseInt(i > 0 ? size.substring(0, i) : size) / 10;
         StringBuilder buf = new StringBuilder(30);
-        for (int j = 0; j < i; j++)
+        for (int j = 2; j <= i; j++)
```

Comment:
do I even want to know?

Reply:
You were too fast, I was about to comment on this hack to ask for advice. This function is use to pad the icon column names so that the letter is positioned above the icon. It's a weird one that I did not know how to fix:

Old:
<img width="111" alt="Captura de pantalla 2020-07-08 a las 9 53 54" src="https://user-images.githubusercontent.com/5738588/86892840-12a2ab00-c101-11ea-92a9-f218fa6e17ba.png">

New:
<img width="156" alt="Captura de pantalla 2020-07-08 a las 9 55 06" src="https://user-images.githubusercontent.com/5738588/86892890-25b57b00-c101-11ea-87de-d2598f349764.png">

I want to at least put a comment before it is merged.